pkg/controller/migmigration/stage.go

```diff
@@ -489,14 +490,28 @@ func (t *Task) stagePodReport(client k8sclient.Client) (report PodStartReport, e
 			}
 		}
 		if !ready {
+			progress = append(
+				progress,
+				fmt.Sprintf(
+					"Stage Pod %s/%s: Not running. Phase %s",
+					pod.Namespace,
+					pod.Name,
+					pod.Status.Phase))
+			report.started = false
 			if !hasHealthyClaims(&pod) {
 				report.failed = true
 			}
-			return
+		} else {
+			progress = append(
+				progress,
+				fmt.Sprintf(
+					"Stage Pod %s/%s: Running",
```

Comment:
```suggestion
					"Pod %s/%s: Running",
```

Reply:
This will make sure that the progress message follows RE : /([A-Za-z]+ [A-Za-z0-9-]+\/[A-Za-z0-9-]+)*\:* *([A-Za-z0-9 \/\(\)-\.]*)/;